### ISSUE TITLE
Tests fail with recent Crypt::ECB versions

### ISSUE THREAD

**Issue by eserte** [Robertof/perl-net-telnet-netgear#3]
My smoker machines started to report failures in the test suite:

```
Bailout called.  Further testing stopped:  Can't load Net::Telnet::Netgear::Packet::Native
#   Failed test 'require Net::Telnet::Netgear::Packet::Native;'
#   at t/base.t line 23.
#     Tried to require 'Net::Telnet::Netgear::Packet::Native'.
#     Error:  Bareword "PADDING_NONE" not allowed while "strict subs" in use at /home/cpansand/.cpan/build/2016031306/Net-Telnet-Netgear-0.04-YLEjsH/blib/lib/Net/Telnet/Netgear/Packet/Native.pm line 53.
# Compilation failed in require at (eval 15) line 2.
FAILED--Further testing stopped: Can't load Net::Telnet::Netgear::Packet::Native
*** Error code 255
```

A look into recent Crypt::ECB versions show that the PADDING_\* constants are not available anymore.

**Comment by eserte** [comment]
See also https://rt.cpan.org/Ticket/Display.html?id=112976

**Comment by Robertof** [comment]
Thanks for the bug report!

This issue has been fixed with `v0.05` (https://github.com/Robertof/perl-net-telnet-netgear/commit/fff25b1fd269ae0f618ae3a940fb706c1dbcb2ad): https://travis-ci.org/Robertof/perl-net-telnet-netgear/builds/115697630

It will be available on CPAN very soon.
